backend/src/db-utils.js

```
@@ -21,8 +22,10 @@ function getSequelize(dataSource) {
       return models.sequelizeIndexerBackendReadOnly;
     case DS_ANALYTICS_BACKEND:
       return models.sequelizeAnalyticsBackendReadOnly;
+    case DS_TELEMETRY_BACKEND:
+      return models.sequelizeTelemetryBackendReadOnly;
     default:
-      return models.sequelizeLegacySyncBackendReadOnly;
+      return models.sequelizeTelemetryBackendReadOnly;
```

Let's throw an exception here